code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -1,4 +1,130 @@
 
+/******BRUTE******/
+
+/datum/chemical_reaction/trophazole
+	name = "Trophazole"
+	id  = /datum/reagent/medicine/trophazole
+	results = list(/datum/reagent/medicine/trophazole = 3)
+	required_reagents = list(/datum/reagent/copper = 1, /datum/reagent/acetone = 2,  /datum/reagent/phosphorus = 1, /datum/reagent/consumable/enzyme = 1)
+
+/datum/chemical_reaction/sal_acid
+	name = "Salicyclic Acid"
+	id = /datum/reagent/medicine/sal_acid
+	results = list(/datum/reagent/medicine/sal_acid = 1)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/medicine/C2/libital = 1)
+
+/******BURN******/
+
+/datum/chemical_reaction/rhigoxane
+	name = "Rhigoxane"
+	id = /datum/reagent/medicine/rhigoxane
+	results = list(/datum/reagent/medicine/rhigoxane/ = 3)
+	required_reagents = list(/datum/reagent/cryostylane = 3, /datum/reagent/bromine = 1, /datum/reagent/lye = 1, /datum/reagent/consumable/enzyme = 1)
+	required_temp = 47
+	is_cold_recipe = TRUE
+
+/datum/chemical_reaction/oxandrolone
+	name = "Oxandrolone"
+	id = /datum/reagent/medicine/oxandrolone
+	results = list(/datum/reagent/medicine/oxandrolone = 1)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/medicine/C2/aiuri = 1)
```

Comment:
Who would win/make more c3 chems?
A chemist
OR
One cargo boi with a departmental budget ID who orders a shitton of medkits, takes the medipens from them, and then casts the medkits aside (extracting the c3 chems from those medipens with a syringe is optional)